lib/utils.py

```diff
@@ -78,8 +78,9 @@ def get_node_ip():
         that is being used as the default gateway.
         This should work on both MacOS X and Linux."""
         try:
-            default_interface = grep(netstat('-nr'), '-e', '^default', '-e' '^0.0.0.0').split()[-1]
-            my_ip = ifaddresses(default_interface)[2][0]['addr']
+            interface_id = max(list(gateways()['default']))
+            default_interface = [gateways()][0]['default'][interface_id][1]
```

Comment:
[gateways()][0] == gateways() right?

Reply:
Yes, you are correct.  Ill change that, ofc.